src/Sylius/Component/Core/Inventory/Updater/IncreasingQuantityUpdaterInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Inventory\Updater;
+
+use Sylius\Component\Core\Model\OrderInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+interface IncreasingQuantityUpdaterInterface
```

Comment:
Seems redundant, has one method with one argument, same as `DecreasingQuantity...`, why won't you merge them?